tests/test_auto3dseg.py

```diff
@@ -0,0 +1,428 @@
+# Copyright (c) MONAI Consortium
```

Comment:
I would suggest to split the tests into several files, like: "test_analyzer.py", "test_operations.py", etc. Easier for future extension.
@wyli What do you think?

Thanks.

Reply:
It was convenient for me to test everything in one script as it reuses the SetUp() for fake datasets generation. Splitting it into multiple files would be good too. Either way works for me.

Reply:
I think the current way is good, we can add new scripts if there are new components in the future.

Reply:
OK, let's keep in 1 file.

Thanks.